app/facet_selector/facet_selector.controller.js

```diff
@@ -12,6 +12,8 @@
       $scope.transfers = Transfer;
       $scope.tags = Tag;
 
+      $scope.date_regex = '\\d\\d\\d\\d([-\/]\\d\\d)?([-\/]\\d\\d)?';
```

Comment:
Hmmm, right. That somewhat defeats the point of using this as a validator to prevent NaN dates; I'll add validation inside the function too.